src/AspNetCore/Src/proxymodule.cxx

```diff
@@ -130,6 +130,9 @@ CProxyModule::OnAsyncCompletion(
 )
 {
     return m_pHandler->OnAsyncCompletion(
+        pHttpContext,
```

Comment:
the handler already wrapped the httpcontext. So far we don't need notification type info and fPostNotifcation